### ISSUE TITLE
Fix values for HTML lang

### ISSUE THREAD

**Issue by AntoineAugusti** [MTES-MCT/Dossier-Facile-Frontend#385]
The `lang` attributes are hardcoded. For tenants it's `fr` and for owners it's `en`.

The HTML lang attribute should be equal to the current language selected. This will be helpful for screenreaders and SEO purposes.

https://github.com/MTES-MCT/Dossier-Facile-Frontend/blob/3f6c65d95f41e204ffc491060fd05e355957f565/owner/public/index.html#L2

https://github.com/MTES-MCT/Dossier-Facile-Frontend/blob/3f6c65d95f41e204ffc491060fd05e355957f565/tenant/public/index.html#L2

**Comment by mattboll** [comment]
Hello,
Thank you for reporting the problem. the owner repository is not used at all so it won't be fixed anytime soon
For the tenant repository, there is a work in progress to improve language selection and we should change this hardcoded value at the same time (right now french is the only fully supported language)

**Comment by mattboll** [comment]
Hi,
I'm closing the issue as we won't use github to handle our issues.
Rest assured that we will fix this bug in a really short term.
You can still send emails to contact@dossierfacile.fr if you have an issue

Regards,
